src/sentry/static/sentry/app/views/organizationDiscover/sidebar/queryFields.jsx

```diff
@@ -138,8 +149,15 @@ export default class QueryFields extends React.Component {
             disabled={isLoading}
           />
         </Fieldset>
+        <FieldSeparator />
         <Fieldset>{actions}</Fieldset>
-      </div>
+        <DocsSeparator />
+        <DicsoverDocs>
```

Comment:
nit: spelling

Reply:
This needs to actually link to the discover page in the docs

Reply:
oh shoot whoops!